bgpd/bgp_route.c

```diff
@@ -124,6 +124,8 @@ static const struct message bgp_pmsi_tnltype_str[] = {
 };
 
 #define VRFID_NONE_STR "-"
+#define SOFT_RECONFIG_TASK_MAX_PREFIX 25000
+#define SOFT_RECONFIG_TASK_SPLIT_DELAY_MS 50
```

Comment:
why use a timer - why not just split up the work using ordinary 'event' callbacks?

Reply:
I have removed the useless timer.